### PR TITLE
Convert travel history into table

### LINKED ISSUE #2682: BUG: Only the first travel history appears in the eCR Viewer
## Description

Only the first travel history appears in the eCR Viewer

Issue: All travel history is properly saved in the FHIR bundle :), but we are only looking at the first result from our [fhir path](https://github.com/CDCgov/phdi/blob/951a0974c2ba086c94b2a2b580c97ce71cbe600c/containers/ecr-viewer/src/app/services/evaluateFhirDataService.ts#L162-L165). 

## Steps to reproduce

_List out the steps in detail to reproduce the bug_

1. Visit [4aca74f4-c3b7-49ce-a437-1c8594451926](http://localhost:3000/view-data?id=1.2.840.114350.1.13.572.3.7.8.688883.102910)
2. View Social History -> Travel History

## Expected Result

- All travel history appears like in the eiCR html

image

## Actual result

- Only the first travel history appears
image

## Priority

_Provide an estimate for level of impact the bug has today by placing an X in the applicable bracket_

#### Priority
- [ ] **Low priority** _The bug does not have to be fixed immediately. High and medium priority bugs should be addressed first._
- [ ] **Medium priority** _Typically, medium-priority bugs do not affect customers directly and therefore can be fixed in the normal course of testing and development._
- [x] **High priority** _High priority bugs must be addressed immediately. They often affect the app’s functionality and impact customers and their user experience. They must take priority._

## Notes

- Design to provide guidance on how to display multiple entries.
  - Put all Location, Start Date, End Date, Purpose, into a table 
- Ensure that it still works with a single entry travel history

### PR DESCRIPTION
**Pull request by BobanL**
# PULL REQUEST

## Summary
- Evaluate all travel history as a table
- Clean up some warnings in test
  - Update keys so the warning `Warning: Each child in a list should have a unique "key" prop.` would go be resolved.
  - Add mock fetch to clean up error

## Related Issue
Fixes #2682 

## Acceptance Criteria
All travel history appears like in the eiCR html
[4aca74f4-c3b7-49ce-a437-1c8594451926](http://localhost:3000/view-data?id=1.2.840.114350.1.13.572.3.7.8.688883.102910)

## Additional Information
Before: 
image

Now:
image

### CODE REVIEW

File: containers/ecr-viewer/src/app/view-data/components/EvaluateTable.tsx
Comment (BobanL) [inline containers/ecr-viewer/src/app/view-data/components/EvaluateTable.tsx:1]: Fixes missing key as described in tests and http://localhost:3000/view-data?id=1.2.840.114350.1.13.478.3.7.8.688883.230886
Before:
image

Now:
image

File: containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx
Comment (BobanL) [inline containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx:1]: Fixes missing key as described in test and http://localhost:3000/view-data?id=1.2.840.114350.1.13.572.3.7.8.688883.102910
Before:
image


Now:
image

File: containers/ecr-viewer/src/app/api/fhirPath.yml [hunk 1]
```diff
@@ -147,3 +144,9 @@ observationSusceptibility: "valueString"
 
 # Organization
 organizations: "Bundle.entry.resource.where(resourceType = 'Organization')"
+
+# Travel History
```
Comment (mcmcgrath13) [inline containers/ecr-viewer/src/app/api/fhirPath.yml:148]: typically I see the bundle one as the top of the block of these - could we move it down here?

File: containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx [hunk 2]
```diff
@@ -57,8 +57,8 @@ const EcrSummary: React.FC<EcrSummaryProps> = ({
               {"Clinical Sections Relevant to Reportable Condition"}
             </div>
             <div className={"margin-top-0"}>
-              {condition.clinicalDetails.map((item) => (
-                <DataTableDisplay item={item} key={item.title} />
+              {condition.clinicalDetails.map((item, index) => (
```
Comment (mcmcgrath13) [inline containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx:60]: I also fixed these in my PR - we'll see who merges first 🐎

**Review by mcmcgrath13 — APPROVED** [review]
LGTM!

File: containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx (hunk 2 above)
Comment (BobanL) [reply containers/ecr-viewer/src/app/view-data/components/EcrSummary.tsx:60]: 😆 Well you were faster to review my pr so I'm going to say this change is going to make it so we all win.

**Pull request merged**